queue_services/activity-log-listener/README.md

```diff
@@ -0,0 +1,75 @@
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+
+
+# Application Name
+
+BC Registries Names Examination, research and approval system API
```

Comment:
I think all the q service is having the Namex readme :)

Reply:
fixed a bit